### DISCUSSION TITLE
Images/CSS wrong in build

### DISCUSSION THREAD

**Discussion by antchris2** [alshedivat/al-folio#1946] [Q&A]
I forked and deployed the base repo just to get it working. However, while everything looks fine locally, when I deploy it, the CSS is quite mangled and the images all do not work. Any advice on what is happening and how to fix it?

**Comment by george-gca** [comment]
Have you fixed this already?

**Reply by antchris2** [reply]
Yes, issue was simply not stating URL correctly in config file. Thanks all!
